### ISSUE TITLE
HW 3 Review

### ISSUE THREAD

**Issue by mkcomer237** [rickjunli/DAT_SF_16#5]
I like how you used urllib to load the data directly from the website (I downloaded it to a csv file first).  

For KNN, I don't understand why you use the first approach of getting the indicices and then using loc to pull out parts of the dataset.  Doesn't train test split already create the arrays you need to fit the model? 

I'm interested to see that you got a pretty good accuracy from KNN even without normalizing the data (I normalized mine and got ~75% accuracy).  It's also good to see the sklearn cross_val_score function in action, I didn't have time to figure this out so I just used the function we built in the lab.

**Comment by rickjunli** [comment]
Thanks @mkcomer237 . 
When I first did my homework, I had trouble getting the .fit function to directly accept data from the data frame, so I ended up trying to access the data using the indexes. I found out yesterday that the function can directly take input from the data frame. Not sure what I did wrong before. I guess this is part of the growing pain. 
@ghego @breucopter Do we need to normalize data for KNN?

**Comment by ghego** [comment]
hey good job with HW!

you can choose different ways to normalize the data. Two standard ways are min_max_scaler and standard_scaler. Look them up on scikit-learn doc.

You can use confusion matrix to calculate FP, TP etc.

keep rocking!

**Comment by ghego** [comment]
Also, did you notice the 0 values? Anything to do about them?
